integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/common/BenchUtils.scala

```diff
@@ -202,25 +210,35 @@ object BenchUtils {
         val end = System.nanoTime()
         val elapsed = NANOSECONDS.toMillis(end - start)
         queryTimes.append(elapsed)
-        println(s"$logPrefix Iteration $i took $elapsed msec.")
+
+        val failureOpt = taskFailureListener.taskFailures.headOption
+        val status = failureOpt.map(_ =>  STATUS_COMPLETED_WITH_TASK_FAILURES)
+            .getOrElse(STATUS_COMPLETED)
+        failureOpt.foreach(failure => exceptions.append(failure.toString))
```

Comment:
sorry if this comment comes through twice, you changed as I was commenting:

do we need to/want to change the suffix on the file at all? I think adding this will make the file name show up with -failed right?  line 257 below

Reply:
That's a good point. Now that we are recording the status per run it probably doesn't make sense to have the convention of adding `-failed` to the filename. What do you think @abellina (I think you may have asked for this feature?).

Reply:
I went ahead and pushed a change to remove the `-failed` suffix.

Reply:
I'm fine either way, if we keep it might want to change it to not be failed if query succeeded but had failures